setup.py

```diff
@@ -243,21 +246,30 @@ def run(self):
             MakePykeRules._pyke_rule_compile()
 
 
+def extract_version():
+    version = None
+    with open('lib/iris/__init__.py') as fd:
+        for line in fd.readlines():
+            if (line.startswith('__version__')):
+                version = eval('dict({})'.format(line.strip()))['__version__']
```

Comment:
I have to admire your lateral thinking with the dict creation, but it'd be nice to avoid using `eval`. Perhaps:
```python
_, version = line.split()
version = version.strip()[1:-1]  # Remove quotation characters.
```